src/Swarm/Game/Scenario/EntityFacade.hs

```diff
@@ -0,0 +1,24 @@
+{-# LANGUAGE DerivingVia #-}
+
+module Swarm.Game.Scenario.EntityFacade where
+
+import Control.Lens hiding (from, (.=), (<.>))
+import Data.Text (Text)
+import Data.Yaml as Y
+import Swarm.Game.Display (Display)
+import Swarm.Game.Entity qualified as E
+
+type EntityName = Text
+
+data EntityFacadeData = EntityFacadeData EntityName Display
```

Comment:
Why not just name it `EntityFacade` as the module name suggests?

Reply:
Also, do you need the whole `Display`? I would expect only needing a `Char`.

Reply:
Turns out I need to use the `Display` [here](https://github.com/swarm-game/swarm/pull/873/files#r1053872827).